app/forms/concerns/hyku_addons/work_form.rb

```diff
@@ -14,8 +14,10 @@ def contributor_list
 
     class_methods do
       def common_fields
-        [:title, :resource_type, :alternative_name, :project_name, :institution, :abstract, creator_fields, contributor_fields,
-         date_published_fields, funder_fields]
+        [:title, :resource_type, :alternative_name, :project_name, :institution, :abstract, :official_link, :related_url,
+         :language, :license, :rights_statement, :rights_holder, :doi, :peer_reviewed, :keywords, :dewey, :library_of_congress_classification,
```

Comment:
I think `:peer_reviewed` is called `:refereed` in the model.  According to the spreadsheet it isn't common to all work types so we can drop it from this list.